lib/inputstreamhelper.py

```diff
@@ -519,10 +559,10 @@ def _install_widevine_x86(self):
     def _install_widevine_arm(self):
         """Installs Widevine CDM on ARM-based architectures."""
         root_cmds = ['mount', 'umount', 'losetup', 'modprobe']
-        cos_config = self._chromeos_config()
-        device = [x for x in cos_config if config.CHROMEOS_ARM_HWID in x['hwidmatch']][0]
-        required_diskspace = int(device['filesize']) + int(device['zipfilesize'])
-        if xbmcgui.Dialog().yesno(LANGUAGE(30001),
+        devices = self._chromeos_config()
+        arm_device = self._select_best_chromeos_image(devices)
+        required_diskspace = int(arm_device['filesize']) + int(arm_device['zipfilesize'])
```

Comment:
It's possible that `arm_device` is `None`, so we better exit this function before doing stuff with a non existing object.

Reply:
Agreed. But that was an issue before as well.

Reply:
This shouldn't happen, because we do not get to this point if we couldn't get a version from recovery.conf (using the same mechanism).